challenges/2021_CyrilMorcrette_cloudBaseHeight/cbh_challenge_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<p>I'd recommend only calling the scratch environment variable once, creating something like <code>root_data_dir=os.environ['SCRATCH']/cbh_data</code> or similar,  and then reference that subsequently, like <code>paths_to_load=root_data_dir </code> + '/dev/dev_large.nc'.</p><p>The advantage of this is that subsequently if someone is running on a different system (e.g. cloud), they would only need to change ideally one line of code to adapt to the new environment, rather than multiple lines.</p><p>I'd also suggest using the pathlib library, as this is the recommended pythonic way of dealing with files, soi it would be something like</p><pre class="ql-syntax" spellcheck="false">root_data_dir = pathlib.Path(os.environ['SCRATCH']) / 'cbh_data'
paths_to_load = root_data_dir / 'dev' / 'dev_large.nc'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>